### ISSUE TITLE
Outdated Dependencies

### ISSUE THREAD

**Issue by itsdanielmatos** [philcockfield/storybook-host#41]
Storybook just released version 4️⃣and it's awesome :100:. This project has two dependencies that could be updated to their latest versions (major versions / breaking changes):

```shell
Package                Current Wanted  Latest  Package Type    URL
@storybook/addon-knobs 3.4.11  3.4.11  4.0.2   dependencies    https://github.com/storybooks/storybook/tree/master/addons/knobs
@storybook/react       3.4.11  3.4.11  4.0.2   devDependencies https://github.com/storybooks/storybook/tree/master/app/react
```

There are also some dev dependencies that could be updated to their latest versions (minor versions / non breaking changes)

```shell
Package                Current Wanted  Latest  Package Type    URL
@types/chai            4.1.5   4.1.7   4.1.7   devDependencies https://github.com/DefinitelyTyped/DefinitelyTyped.git
@types/node            10.11.0 10.12.2 10.12.2 devDependencies https://github.com/DefinitelyTyped/DefinitelyTyped.git
@types/ramda           0.25.38 0.25.38 0.26.0  devDependencies https://github.com/DefinitelyTyped/DefinitelyTyped.git
@types/react           16.4.14 16.4.18 16.4.18 devDependencies https://github.com/DefinitelyTyped/DefinitelyTyped.git
@types/react-dom       16.0.7  16.0.9  16.0.9  devDependencies https://github.com/DefinitelyTyped/DefinitelyTyped.git
react                  16.5.2  16.6.0  16.6.0  devDependencies https://reactjs.org/
react-dom              16.5.2  16.6.0  16.6.0  devDependencies https://reactjs.org/
typescript             3.0.3   3.1.6   3.1.6   devDependencies http://typescriptlang.org/
```

**PS:** I am already working on this issue, just wanted to open an issue so that the pull request has some context.

**Comment by philcockfield** [comment]
Ah great - they've done the release.

>>PS: I am already working on this issue, just wanted to open an issue so that the pull request has some context.

Nice one! Thanks @itsdanielmatos 👍 ....you beauty!

**Comment by philcockfield** [comment]
>> There are also some dev dependencies that could be updated to their latest versions (minor versions / non breaking changes)

yes please, updating those other deps too would be much appreciated.

**Comment by itsdanielmatos** [comment]
A PR has been created #42 :tada:

**Comment by itsdanielmatos** [comment]
Closed by #42.

**Comment by itsdanielmatos** [comment]
Closed by #42

**Comment by itsdanielmatos** [comment]
Closed by #42.
